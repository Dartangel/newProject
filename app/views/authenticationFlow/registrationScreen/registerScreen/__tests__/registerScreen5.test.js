import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import RegistrationScreen5 from '../registerScreen5';

describe('RegistrationScreen5', () => {
    test('RegistrationScreen5 snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<RegistrationScreen5/>);
        expect(snapshot).toMatchSnapshot();
    });
});