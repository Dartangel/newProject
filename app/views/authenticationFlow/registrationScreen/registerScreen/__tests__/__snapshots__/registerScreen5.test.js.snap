// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationScreen5 RegistrationScreen5 snapshot 1`] = `
<View>
  <TouchableOpacity
    activeOpacity={0.2}
    onPress={[Function]}
  >
    <Text>
       Вход 
    </Text>
  </TouchableOpacity>
</View>
`;
